src/app/GitUI/CommandsDialogs/RepoHosting/CreatePullRequestForm.cs

```diff
@@ -90,6 +92,29 @@ private void LoadRemotes(IHostedRemote[] foreignHostedRemotes)
             _pullReqTargetsCB_SelectedIndexChanged(this, EventArgs.Empty);
         }
 
+        private void LoadPRTemplate()
+        {
+            string templatePath = Path.Combine(Module.WorkingDir, ".github", "PULL_REQUEST_TEMPLATE.md");
+
+            if (File.Exists(templatePath))
+            {
+                ThreadHelper.FileAndForget(async () =>
+                {
+                    try
+                    {
+                        string template = await File.ReadAllTextAsync(templatePath);
+
+                        await this.SwitchToMainThreadAsync();
+                        _bodyTB.Text = template;
+                    }
+                    catch (Exception ex)
+                    {
+                        Trace.TraceError($@"Failed to read pull request template ""{templatePath}"": {ex}");
```

Comment:
I don't believe this is common in our codebase. The error here is likely be a user-induced error, and it is something the user will need to fix. So, we have to show this to the user. For this, IIRC, we should throw `UserExternalOperationException` (wrapping `ex` in `ExternalOperationException`).
It would be good to test this path, at least manually.